mycroft/configuration/mycroft.conf

```diff
@@ -392,11 +392,12 @@
     "multiplier": 1.0,
     "energy_ratio": 1.5,
 
-    // DEPRECATED, multiple hotwords are supported now, see "hotwords" section below
-    // wake_word can be used to get a speakable string of main wake word, ie, mycrofts name
+    // NOTE, multiple hotwords are supported now, these fields define the main wake_word,
+    // this is equivalent to setting "active": true in the "hotwords" section below IF "active" is missing
+    // this field is also used to get a speakable string of main wake word, ie, mycrofts name
     // this is set by selene and used in naptime skill
     "wake_word": "hey_mycroft",
-    //"stand_up_word": "wake_up",
+    "stand_up_word": "wake_up",
```

Comment:
Should this be present by default? "wake up" behavior isn't core functionality

Reply:
it is core functionality indeed, its part of the listener loop and explicitly handled, only thing not core is the voice interface to enable it, but you can do that with a bus event programmatically, eg, lock down the device during the night and unlock in the morning

Reply:
this is keeping same behavior, because active used to default to True, if we dont have this here then "wake up" will be disabled for everyone out of nowhere

this change also allows users to change wake word without having to go into its config to set "active": False, which means we can bundle arbitrary hotword configs for reference. 

The main take away here is that we are no longer enforcing the default word to be enabled, to change wake word a user would need to set "active": False explicitly in our default values